README.org

```diff
@@ -1,17 +1,33 @@
+* Eventually Matrix
+
+  [[https://en.wikipedia.org/wiki/Eventual_consistency][Eventually]] Matrix is a
+  repository of knowledge about the Matrix ecosystem maintained by the community
+  for the community. It is the "bleeding edge" counterpart of the official
+  Matrix.org repositories in terms of documentation, SDK and libraries.
+
 * Purpose                                              :matrix:eventually_matrix:
 
-  Eventually Matrix is a repository of knowledge about the Matrix ecosystem maintained
-  by the community for the community. In broad strokes, this repository exists for
-  the following purposes in roughly the order they're listed.
+  In broad strokes, this repository exists for the following purposes in roughly the
+  order they're listed.
 
+  - Allow Matrix community to easily document the current behavior of implementations
+    in a specification format when such behaviour deviates from the upstream spec.
+    - Speed up the process of adoption and validation of new features/endpoints.
+    - Give a more bleeding-edge reference document that Matrix community can use to
+      prepare for stable releases of the official specification.
+    - Avoid duplication of work and effort in reverse-engineering deviation from
+      outdated specification.
   - Allow Matrix community to easily document pieces of the de-facto Matrix spec
```

Comment:
Should this be merged with new point introduced above?

Reply:
Not in my mind as they target different use cases:
- First point is about documenting the current implementations, regardless of how good those are. It allows to build something that works with the current ecosystem.
- Second point is documenting points that the Matrix.org team agreed on and might label an existing implementation behavior as buggy. This would happen after the first point, once  you've documented something and discussed it with the Matrix team.